apps/opencs/view/render/terraintexturemode.cpp

```diff
@@ -34,14 +36,18 @@
 #include "pagedworldspacewidget.hpp"
 #include "mask.hpp"
 #include "object.hpp" // Something small needed regarding pointers from here ()
+#include "terrainselection.hpp"
```

Comment:
Since terraintexturemode.hpp already depends on this header, this include is redundant.